packages/p2p/src/components/modal-manager/modals/quick-add-modal/quick-add-modal-content/add-payment-methods/add-payment-methods.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { observer } from '@deriv/stores';
+import { useStores } from 'Stores';
+import { Localize } from 'Components/i18next';
+import AddPaymentMethod from 'Pages/my-profile/payment-methods/add-payment-method/add-payment-method.jsx';
+import SellAdPaymentMethodsList from 'Pages/my-ads/sell-ad-payment-methods-list.jsx';
+
+type TAddPaymentMethodsProps = {
+    selected_methods: string[];
+    setSelectedMethods: React.Dispatch<React.SetStateAction<string[]>>;
+};
+
+const AddPaymentMethods = ({ selected_methods, setSelectedMethods }: TAddPaymentMethodsProps) => {
+    const { my_ads_store } = useStores();
+
+    const { payment_method_ids, should_show_add_payment_method, setShouldShowAddPaymentMethod } = my_ads_store;
+
+    const onClickPaymentMethodCard = (payment_method: { ID: string }) => {
+        if (!payment_method_ids.includes(payment_method.ID)) {
+            if (payment_method_ids.length < 3) {
+                my_ads_store.payment_method_ids.push(payment_method.ID);
+                setSelectedMethods([...selected_methods, payment_method.ID]);
+            }
+        } else {
+            my_ads_store.payment_method_ids = my_ads_store.payment_method_ids.filter(
+                (payment_method_id: string) => payment_method_id !== payment_method.ID
+            );
+            setSelectedMethods(selected_methods.filter((i: string) => i !== payment_method.ID));
+        }
+    };
+
+    if (should_show_add_payment_method) {
+        return <AddPaymentMethod should_show_page_return={false} should_show_separated_footer={true} />;
```

Comment:
will this work?
`<AddPaymentMethod should_show_page_return={false} should_show_separated_footer />;`